### PR TITLE
Added timer

### LINKED ISSUE #2: Issue 2: Add index.html and script.js
Add an index.html file that would be shown on the newtab page (refer to issue #1). 

The webpage should have a working timer shown prominently. Their should be a function in script.js to easily set this timer to any date and time.

### PR DESCRIPTION
**Pull request by criticic**
Issue: 2

#### Short description of what this resolves:

Adds a timer

#### Changes proposed in this pull request and/or Screenshots of changes:
Screenshot from 2021-10-04 21-59-57

Resolves #2

### CODE REVIEW

**Comment by criticic** [comment]
@rag-hav Please take a look.

File: chrome/scripts.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+var targetDate = new Date("Nov 4, 2021").getTime();
```
Comment (rag-hav) [inline chrome/scripts.js:1]: One of the requirement of the issue is to have a function that can change the timer's targetDate. Also please avoid making global variables as many people will be working on this code, and polluting the global namespace will make the code harder to understand for them.

**Review by rag-hav — CHANGES_REQUESTED** [review]
Just one problem with the PR. Rest everything is good.

File: chrome/scripts.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    var time = remainingTime(date);
+    document.getElementById("timer").innerHTML = time.days + "d " + time.hours + "h " + time.minutes + "m " + time.seconds + "s Left";
+
+    if (time.totalInterval <= 0) {
+        clearInterval(x);
+        document.getElementById("timer").innerHTML = "Contest has Started";
+    }
+}
+
+function runClock() {
```
Comment (rag-hav) [inline chrome/scripts.js:28]: give this function the targetDate as its argument and we are golden!

**Review by rag-hav — CHANGES_REQUESTED** [review]
just a one line change required

File: chrome/scripts.js (hunk 2 above)
Comment (criticic) [reply chrome/scripts.js:28]: `function runClock(targetDate) {
    var targetDate = new Date("Nov 4, 2021").getTime();
    setClock(targetDate);
}` like this?

Comment (rag-hav) [reply chrome/scripts.js]: No the function should take in its argument, the target time of the timer. 

like runClock(new Date("Nov 10, 2020"));

**Pull request merged**